src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/UserManagement/PrereleaseUsersController.cs

```diff
@@ -0,0 +1,38 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces;
+using GovUk.Education.ExploreEducationStatistics.Admin.ViewModels;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Controllers.Api.UserManagement
+{
+    [Route("api")]
+    [ApiController]
+    [Authorize(Policy = "CanManageUsersOnSystem")]
+    public class PrereleaseUsersController : ControllerBase
+    {
+        private readonly IUserManagementService _userManagementService;
+
+        public PrereleaseUsersController(IUserManagementService userManagementService)
+        {
+            _userManagementService = userManagementService;
+        }
+
+        [HttpGet("user-management/pre-release")]
```

Comment:
Feels a bit wordy to be prefixing everything with `user-management`. Why don't we simply name it `users`?

Reply:
Was aligning this with the namespaces and something common that describes what these split controllers do. Did have users earlier on but it didn't really work as a nice base given theres /users path so i added -management.

Reply:
Wouldn't say there's any reason why controllers that share the same namespace in code need to share the same prefix in the API. I think the controllers being split up is good stuff, but tacking on the prefix just feels unnecessarily verbose and is a little bit anti-REST.

From what I can see, the endpoints could just be:
- `/users`
- `/pre-release-users` (or even better - `/api/users?type=pre-release`
- `/invites`
- `/roles`
- `/release-roles`
- `/releases` - isn't there already a releases endpoint for the rest of admin???

Reply:
I'm not too concerned by the prefix so not intending to change it.

Reply:
The logic and contents of new new release endpoint are quite specific to user management where a simplified fill list is required regardless of any access permissions. The exact need implementation of this will probably change when the next user management and permissions epic starts.